### PR TITLE
tinysearch 0.7.0

### PR DESCRIPTION
**Pull request by ms-ati**
* Fix tests for the "wasm_output" path added in v0.7.0

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
This will need an official release: https://github.com/tinysearch/tinysearch/issues/164

**Comment by bayandin** [comment]
The tag has been created, need to update the PR to use it.

File: Formula/tinysearch.rb [hunk 1]
```diff
@@ -30,7 +31,7 @@ def install
     ENV.prepend_path "PATH", HOMEBREW_CACHE/"cargo_cache/bin"
 
     system bin/"tinysearch", pkgshare/"fixtures/index.json"
-    assert_predicate testpath/"tinysearch_engine_bg.wasm", :exist?
-    assert_match "A tiny search engine for static websites", (testpath/"package.json").read
+    assert_predicate testpath/"wasm_output"/"tinysearch_engine_bg.wasm", :exist?
+    assert_match "A tiny search engine for static websites", (testpath/"wasm_output"/"package.json").read
```
Comment (carlocab) [inline Formula/tinysearch.rb:35]:
```suggestion
    assert_predicate testpath/"wasm_output/tinysearch_engine_bg.wasm", :exist?
    assert_match "A tiny search engine for static websites", (testpath/"wasm_output/package.json").read
```

File: Formula/tinysearch.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
 class Tinysearch < Formula
   desc "Tiny, full-text search engine for static websites built with Rust and Wasm"
   homepage "https://github.com/tinysearch/tinysearch"
-  url "https://github.com/tinysearch/tinysearch/archive/refs/tags/v0.6.0.tar.gz"
-  sha256 "a166ea3f0a5f0bfd490cb36b3360b51e40a278e138f40a4be601faf4d33f456b"
+  url "https://github.com/tinysearch/tinysearch/archive/5d98c355e4f6a0e8c7f67e5764cd0322883ecdef.tar.gz"
+  version "0.7.0"
```
Comment (SMillerDev) [inline Formula/tinysearch.rb:5]:
```suggestion
  url "https://github.com/tinysearch/tinysearch/archive/refs/tags/v0.7.0.tar.gz"
```

Comment (ms-ati) [reply Formula/tinysearch.rb:5]: Don’t we still need the “version” line?

Comment (SMillerDev) [reply Formula/tinysearch.rb:5]: No, brew extrapolates it from the URL

Comment (ms-ati) [reply Formula/tinysearch.rb:5]: Oh, thank you!

**Review by SMillerDev — DISMISSED** [review]

**Comment by chenrui333** [comment]
Not sure why the forced push accidentally closed this PR, I will create a new one instead.

**Comment by chenrui333** [comment]
#115886

**Pull request closed without merging**